src/Compute/v2/Api/Server.php

```diff
@@ -0,0 +1,204 @@
+                'networks' => [
+                    'type' => 'array',
+                    'items' => [
+                        'type' => 'object',
+                        'items' => [
+                            'uuid' => ['type' => 'string'],
+                            'port' => ['type' => 'string'],
+                        ]
+                    ]
+                ],
+                'name' => ['type' => 'string', 'required' => true],
+                'metadata' => ['type' => 'string'],
+                'personality' => ['type' => 'string'],
+                'blockDeviceMapping' => [
+                    'type' => 'array',
+                    'sentAs' => 'block_device_mapping_v2',
+                    'items' => [
+                        'type' => 'object',
+                        'items' => [
+                            'configDrive' => ['type' => 'string', 'sentAs' => 'config_drive'],
+                            'bootIndex' => ['type' => 'string', 'sentAs' => 'boot_index'],
+                            'deleteOnTermination' => ['type' => 'boolean', 'sentAs' => 'delete_on_termination'],
+                            'guestFormat' => ['type' => 'string', 'sentAs' => 'guest_format'],
+                            'destinationType' => ['type' => 'string', 'sentAs' => 'destination_type'],
+                            'sourceType' => ['type' => 'string', 'sentAs' => 'source_type'],
+                            'deviceName' => ['type' => 'string', 'sentAs' => 'device_name'],
+                        ]
+                    ],
+                ],
+            ]
+        ];
+    }
+
+    public static function getAll()
+    {
+        return [
+            'method' => 'GET',
+            'path'   => 'servers',
+            'params' => [
+                'changesSince' => ['sentAs' => 'changes-since', 'type' => 'string', 'location' => 'query'],
+                'imageId'      => ['sentAs' => 'image', 'type' => 'string', 'location' => 'query'],
+                'flavorId'     => ['sentAs' => 'flavor', 'type' => 'string', 'location' => 'query'],
+                'name'         => ['type' => 'string', 'location' => 'query'],
+                'marker'       => ['type' => 'string', 'location' => 'query'],
+                'limit'        => ['type' => 'string', 'location' => 'query'],
+                'status'       => ['type' => 'string', 'location' => 'query'],
+                'host'         => ['type' => 'string', 'location' => 'query']
+            ],
+        ];
+    }
+
+    public static function getAllDetail()
+    {
+        $definition = self::getServers();
+        $definition['path'] += '/detailed';
+        return $definition;
+    }
+
+    public static function get()
+    {
+        return [
+            'method' => 'GET',
+            'path'   => 'servers/{serverId}',
+            'params' => [
+                'id' => ['type' => 'string', 'required' => true, 'location' => 'url']
```

Comment:
Should this `id` really be `serverId` (or vice versa)? Otherwise I'm confused how/where they are being mapped to each other.